README.md

```diff
@@ -12,9 +17,7 @@ with no central authority: managing transactions and issuing money are carried
 out collectively by the network. Bitcoin Core is the name of open source
 software which enables the use of this currency.
 
-For more information, as well as an immediately usable, binary version of
-the Bitcoin Core software, see https://bitcoincore.org/en/download/, or read the
-[original whitepaper](https://bitcoincore.org/bitcoin.pdf).
+For more information read the original Bitcoin whitepaper.
```

Comment:
isn't that a bit flimsy? It's not so hard to game Google rankings (and you can always buy ads, too), so googling "original Bitcoin whitepaper" could really give you anything. At least a footnote or something with bibliographic information (such as it is) would be nice, or maybe a md5sum of the "real" whitepaper, if that is feasible

Reply:
Crazy idea: what about adding instructions (or a script) on how to fetch the whitepaper directly from the UTXO set, as described e.g. by jb55 in https://bitcoinhackers.org/@jb55/105595146491662406 ("eat your own dogfood" ;-))

Reply:
> or maybe a md5sum of the "real" whitepaper, if that is feasible

Inb4 content-addressed storage (e.g. IPFS).

In any case, something like that could be added later, let's remove the broken link.

Reply:
Maybe there could be an extensive stack exchange question or documentation file that explains where and how to get the whitepaper and how to check it against a hashsum? If needed we could link to that. Though, I think this can be done in a follow-up. And if your search engine was bought to redirect the traffic, you might not find this repo in the first place ;)

Reply:
Apparently it is actually on IPFS. [gateway link here](https://gateway.ipfs.io/ipfs/QmRA3NWM82ZGynMbYzAgYTSXCVM14Wx1RZ8fKP42G6gjgj)
Users of [some browsers](https://github.com/ipfs/in-web-browsers) like Brave can use the IPFS link directly and don't  need the gateway.

Reply:
Would this suffice as the Stack Exchange question to link to? https://bitcoin.stackexchange.com/questions/35959/how-is-the-whitepaper-decoded-from-the-blockchain-tx-with-1000x-m-of-n-multisi

Reply:
@michaelfolkson sort of ! unfortunately it uses the blocktrail API instead of bitcoin core RPC, i prefer to send people on a puzzle to get it and learn to use our APIs along the way  :smile:

Reply:
There's a comment on how to do it with a pruned node, using the RPC: https://bitcoin.stackexchange.com/questions/35959/how-is-the-whitepaper-decoded-from-the-blockchain-tx-with-1000x-m-of-n-multisi#comment115247_101707